### PR TITLE
Create README file

### PR DESCRIPTION
**Pull request by alexdqmf**
Alongside a change to gitignore (please check).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,7 +22,7 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
-target/
```
Comment (alextopher0) [inline .gitignore]: Is this not correct https://stackoverflow.com/questions/343646/ignoring-directories-in-git-repos-on-windows?

**Comment by DevOrc** [comment]
You should probably add a section on [how to install maven](https://maven.apache.org/install.html). Otherwise this looks good!

Also this should probably be squashed when merged...

**Comment by RedRoyalDog** [comment]
We can add the maven thing later :)

**Comment by suppergerrie2** [comment]
Wait you need to install maven? I haven't had the need to do that? And with gradle you can just distribute the wrapper in repo and you do not need to install it, there isn't something like that for maven?

**Pull request merged**